bazel/foreign_cc/vectorscan.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/src/util/arch/arm/cpuid_inline.h b/src/util/arch/arm/cpuid_inline.h
+index 03faf41..48dc6d0 100644
+--- a/src/util/arch/arm/cpuid_inline.h
++++ b/src/util/arch/arm/cpuid_inline.h
+@@ -31,6 +31,7 @@
+ #define AARCH64_CPUID_INLINE_H_
+ 
+ #if defined(__linux__)
++#include <asm/hwcap.h>
```

Comment:
Can you fix this upstream?

Reply:
opened https://github.com/VectorCamp/vectorscan/issues/176 waiting for response on ISSUE-1 there

Reply:
https://github.com/VectorCamp/vectorscan/issues/180

Reply:
Will this be fixed before we merge?

Reply:
You may want to wait a few days, it will be included yes, but I'm fixing clang 15/16 issues right now as they are higher priority and ironing out a few issues with the new CI. I'll post an update here when that happens.